FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/MessageDetailsFragment.kt

```diff
@@ -1381,6 +1383,16 @@ class MessageDetailsFragment : BaseFragment(), ProgressBehaviour, View.OnClickLi
     })
   }
 
+  private fun isHiddenAtt(it: AttachmentInfo): Boolean {
+    return when {
```

Comment:
could be method on `AttachmentInfo` and simplified to:
```kotlin
public fun isHidden() = name.isNullOrEmpty() && type.lowercase() == "application/pgp-encrypted; name=\"\""
```

Reply:
I've added changes. Please re-check